code/modules/cargo/packs.dm

```diff
@@ -419,9 +425,10 @@
 
 /datum/supply_pack/security/armory/energy
 	name = "Energy Guns Crate"
-	desc = "Contains two Energy Guns, capable of firing both nonlethal and lethal blasts of light. Requires Armory access to open."
-	cost = 2500
+	desc = "Contains three Energy Guns, capable of firing both nonlethal and lethal blasts of light. Requires Armory access to open."
+	cost = 2750
```

Comment:
That's a pretty minor price increase for a 1.5x increase in total firepower

Reply:
Thats now how 1.5x works